web/srv/api_handlers.go

```diff
@@ -434,11 +434,44 @@ func (h *handler) handleAPIResourceDefinition(w http.ResponseWriter, req *http.R
 	w.Write(resourceDefinition)
 }
 
-func (h *handler) handleGetExtension(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+func (h *handler) handleGetExtensions(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
 	ctx := req.Context()
 	extensionName := req.FormValue("extension_name")
 
 	resp := map[string]interface{}{}
+	if extensionName == "" {
+		extensions, err := h.k8sAPI.GetAllNamespacesWithExtensionLabel(ctx)
+		if err != nil {
+			renderJSONError(w, err, http.StatusInternalServerError)
+			return
+		}
+
+		type Extension struct {
+			Name string `json:"name"`
+			UID  string `json:"uid"`
```

Comment:
What is the purpose of the `UID` here?

Reply:
On the react side, the table needs a unique key while iterating through the values - that's why I decided to add the UID instead of generating a new id since its part of the info